.vscode/settings.json

```diff
@@ -1,5 +1,7 @@
 {
     "python.linting.pylintEnabled": true,
     "python.linting.enabled": true,
-    "python.venvPath": "${workspaceFolder}/.venv/bin/python"
+    "python.venvPath": "${workspaceFolder}/.venv/bin/python",
+    "python.pythonPath": "C:\\edfi\\Ed-Fi-X-Fizz\\experimental\\canvas\\explorations-py\\.venv",

```

Comment:
Can this be changed to use `${workspaceFolder}`?

Reply:
Yes, I have updated it.